examples/static/source/Root.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Link } from 'react-router'
 
 const addBase = (path) => (
-  `${(process.env.BASE_PATH || '').replace(/\/$/, '') || '/'}${path}`
+  `${(process.env.BASE_PATH || '').replace(/\/$/, '') || '/'}${path.slice(1)}`
```

Comment:
Since we're on the server, this can be simplified a bit with the `path` module. `path.join('/', '/', '/something')` resolves to `/something` - it already handles for empty strings or just plain slashes. In general you don't have to worry about carefully trimming off leading slashes or adding them.